apyros/test_metalog.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from metalog import *
+
+class MetaLogTest(unittest.TestCase):
+
+    def test_missing_remission(self):
+        # TODO handle relative paths to test
+        ml = MetaLog('test_data/meta_170516_192926.log')
```

Comment:
Please use file path relative to the __file__ entry so it does not depend on the value of the current directory.